app/services/spree_avatax_official/create_tax_adjustments_service.rb

```diff
@@ -3,23 +3,26 @@ class CreateTaxAdjustmentsService < SpreeAvataxOfficial::Base
     include SpreeAvataxOfficial::TaxAdjustmentLabelHelper
 
     def call(order:) # rubocop:disable Metrics/AbcSize
-      return failure(::Spree.t('spree_avatax_official.create_tax_adjustments.order_canceled')) if order.canceled?
+      order.save!
```

Comment:
Without this save checkout throw error:
`Locking a record with unpersisted changes is not supported. Use save to persist the changes, or reload to discard them explicitly.`